test/standalone/main.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable id-length */
+// @flow
+
+const webpack = require('webpack');
+const requireFromString = require('require-from-string');
+const MemoryFS = require('memory-fs');
+const path = require('path');
+const config = require('./webpack.config');
+
+const fs = new MemoryFS();
```

Comment:
why is this used?

Reply:
This is the setup for the standalone webpack server and client, which run from memory saves a lot of pollution of output files